docs/migration.md

```diff
@@ -24,6 +24,10 @@ license: |
  - Celeborn 0.2 Client is compatible with 0.3 Master/Server, it allows to upgrade Master/Worker first then Client.
    Note that: It's strongly recommended to use the same version of Client and Celeborn Master/Worker in production.
 
+ - Since 0.3.0, the support of deprecated configurations `rss.*` is removed.
+   All configurations listed in 0.2.1 docs still take affects, but some of those are deprecated too, please read
+   the bootstrap logs and follow the suggestion to migrate to the new configuration.
```

Comment:
I believe this is achieved by CELEBORN-601 

@AngersZhuuuu, could you confirm that?

Reply:
Yea